### ISSUE TITLE
0.2.9 Animation toggle - Initial state appears flipped

### ISSUE THREAD

**Issue by Garoun** [Frick/ttplus#29]
When loading with Site Layout enabled (haven't tested otherwise) the Animation toggle is using the 'no animation' icon instead of the 'animation' one.  Should probably be loading with the Animation icon.

**Comment by Frick** [comment]
Agreed.  The same goes for the "Enable / Disable Scripts" button, but so far that only affects one room.  I'm not sure simply flipping it is really the best solution.  There simply needs to be a better way of indicating what state the toggle is in.

**Comment by Garoun** [comment]
I agree with your suggestion.
